tests/run-pass/function_calls/exported_symbol.rs

```diff
@@ -15,26 +15,20 @@ fn baz() -> i32 {
     -3
 }
 
-// Make sure shims take precedence.
-#[no_mangle]
-extern "C" fn exit(_: i32) -> ! {
-    unreachable!()
-}
-
-#[no_mangle]
-extern "C" fn ExitProcess(_: u32) -> ! {
-    unreachable!()
-}
-
```

Comment:
I'm sorry that I was wrong in https://github.com/rust-lang/miri/pull/1776#discussion_r615233331: these two function seems meaningless -- no "found `...` symbol definition that clashes with a built-in shim" errors are reported for them. I guess the `start` lang item just returns normally and they are not called at all...

Reply:
No worries. :)